client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -1279,5 +1345,35 @@ describe('FeedbackGeneratorService', function() {
       }
     );
   });
-});
 
+  describe('_getRandomInt', function() {
+    it('should return a random number within the defined range',
+      function() {
+        var min = 0;
+        var max = 4;
+        var maxTries = 100;
+        for (var i = 0; i < maxTries; i++) {
+          var randomNumber = FeedbackGeneratorService._getRandomInt(min, max);
+          expect(typeof randomNumber).toEqual('number');
+          expect(randomNumber).toBeGreaterThan(-1);
```

Comment:
It would probably be a bit stronger if you had "greater than -0.00001" and "less than 3.00001".

Or, better: not.toBeLessThan(0) and similar. See https://stackoverflow.com/questions/24090270/how-can-i-test-that-a-value-is-greater-than-or-equal-to-in-jasmine

Reply:
Yeah, .not. is better, changed.